### DISCUSSION TITLE
TTS model support

### DISCUSSION THREAD

**Discussion by CharlieBickerton** [sonos/tract#815] [Q&A]
Hey @kali , have you or anyone you know deployed TTS models (e.g. fastspeech2) using tract?

**Comment by kali** [comment]
Hello, not that I know of. I would be happy to get some feedback if you give it a shot.

**Reply by CharlieBickerton** [reply]
Okay thanks, I'll let you know if I give it a shot

**Reply by Areopagitics** [reply]
any further news on this? I'm interested as well. I started an issue about kokoros tts where I gave it a shot, but got some errors #1634
